app/mailers/caretaker_notifier_mailer.rb

```diff
@@ -1,7 +1,8 @@
 class CaretakerNotifierMailer < ApplicationMailer
-  def inform(user, friend_contact)
+  def inform(user, friend_contact, garden)
     @user = user
-    mail(to: friend_contact, subject: "#{user.first_name} wants you to check out their gardens!")
+    @garden = garden
+    mail(to: friend_contact, subject: "#{user.first_name} wants you to caretake their garden!")
```

Comment:
`caretake your garden` is a bit awkward - how about `take care of their garden!`?